report/sections/Chapter 1/Intro.tex

```diff
@@ -1 +1,53 @@
-\chapter{P7 Intro}
+\chapter{Introduction} \label{chap:introduction}
+On the 1st semester of the Software Master’s Degree at \aau{} the programming paradigms course currently has a problem where most of the attending students failed the course last time it was held. This led to a large survey being sent out and a student task force being assembled to grasp the main flaws with how the course was structured. Here a large part of the students mentioned a lack of expertise in the programming languages taught in the course and a lack of possibilities for actively training their skills during the semester. 
+
+
+With this knowledge in mind, \aau{} chose to restructure the course with a larger focus on practical programming as opposed to the more classic lectures that are usually used at universities. To support this way of teaching, the course lectures have been moved to a more classical classroom setup but with a monitor at each table and a main monitor at the lecture desk. This is meant to allow the students to more easily work together in groups while working on the shared monitor and allowing the lectures to view and share the groups monitors allowing actual implementations by the students to be used for classroom discussions. 
```

Comment:
~~knowledge~~
"Restructure the course entirely. The course, which previously introduced the students to a large number of languages" 
What is the actual problem? Right now it does not seem that we try to fix a problem but rather accomodate changes.